From: Samuel Brian <samuel.brian@digi.com>
Date: Tue, 26 May 2020 09:52:26 +1000
Subject: build: drop libudev version requirement to at least 182

This libgudev package is compatible with libudev version 182 from the
old udev package at
  https://www.kernel.org/pub/linux/utils/kernel/hotplug/udev-182.tar.bz2
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index a8ec0ec..3d4c613 100644
--- a/configure.ac
+++ b/configure.ac
@@ -38,7 +38,7 @@ GOBJECT_INTROSPECTION_CHECK([1.31.1])
    AM_CONDITIONAL([HAVE_INTROSPECTION], [false])
    enable_introspection=no])
 
-PKG_CHECK_MODULES([LIBUDEV], [libudev >= 199])
+PKG_CHECK_MODULES([LIBUDEV], [libudev >= 182])
 PKG_CHECK_MODULES([GLIB], [glib-2.0 >= 2.38 gobject-2.0 gio-2.0])
 
 AC_ARG_ENABLE([umockdev],
